sns/parity_math.c

```diff
@@ -318,11 +322,7 @@ M0_INTERNAL int m0_parity_math_init(struct m0_parity_math *math,
 		math->pmi_parity_algo = M0_PARITY_CAL_ALGO_XOR;
```

Comment:
Indentation issue for the previous line, No. 321.

Reply:
It had 8 spaces and rest were having tab at the start. Looks like it was issue with github which was not displaying tab as 8 spaces. So it appeared to be indentation issue. I have replaced 8 spaces with tab now.